### PR TITLE
Quiz card and buttons

### PR DESCRIPTION
**Pull request by Tanvez**
The styling isn't there but the buttons work with the cards. Issue #137 
let me know if i should go ahead and change anything.

ezgif-3-adc114fff4c8
ezgif-3-9d26c8f08224

### CODE REVIEW

File: client/src/components/Card.js [hunk 1]
```diff
@@ -23,13 +48,18 @@ const Card = (props) => {
...
+         {mode==='quiz' && <Quiz data={data}/>}
+=======
           <div className="card" style={{textAlign:'left'}}>
             <h1 style={{color:'red', fontStyle:'cursive'}}>{title}</h1>
             <h3 style={{fontSize:'1rem'}}>{body}</h3>
               <footer style={{textAlign:'left'}}>{footer}
                 {link && <Button size="small" href={link} >Learn More</Button>}
               </footer>
           </div>
+>>>>>>> master
```
Comment (SomeMoosery) [inline client/src/components/Card.js]: These `<<<` `===` and `>>>` are from the merge and need to be removed for the code to compile

File: client/src/components/FlashCards.js [hunk 2]
```diff
@@ -4,44 +4,6 @@ import { useDrag } from 'react-use-gesture'
...
-    img:'https://upload.wikimedia.org/wikipedia/en/d/de/RWS_Tarot_01_Magician.jpg',
-    footer:'fun fact# 5'
-  },
-  {
-    title:'Did you know?',
-    body:"In Covid-19,'CO' stands for 'corona', 'VI' for 'virus', and 'D' for disease. Formerly, this disease was referred to as 2019 novel coronavirus or 2019-nCov. There are many types of human coronaviruses including some that commonly cause mild upper-respiratory tract illness",
-    footer:'fun fact# 6'
-  }
-]
-
```
Comment (SomeMoosery) [inline client/src/components/FlashCards.js]: this was meant to move into `fakeCardData.js` right?

Comment (Tanvez) [reply client/src/components/FlashCards.js]: yea i moved it just to make it easier to code out the component

Comment (SomeMoosery) [reply client/src/components/FlashCards.js]: cool cool

File: client/src/components/Card.js (hunk 1 above)
Comment (Tanvez) [reply client/src/components/Card.js]: ahh i just saw that after committing. Thanks!

**Comment by SomeMoosery** [comment]
Awesome job - this looks great! Way better than what I thought we could swing in just a day or two!

**Pull request merged**